src/detection/localip/localip_linux.c

```diff
@@ -27,6 +28,128 @@
 #include <sys/sockio.h>
 #endif
 
+static const NIFlag nicFlags[] = {
+    {
+        .flag = IFF_UP,
+        .name = "UP",
+    },
```

Comment:
I prefer `{ IFF_UP, "UP" }` so that they take less spaces.

Reply:
Done.